src/components/TextField/docs/index.story.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { TextField } from '..';
+import { Examples } from './examples';
+
+export default {
+  category: 'Components',
+  storyName: 'TextField',
+
+  component: TextField,
+  componentPath: '../TextField.tsx',
+
+  componentProps: (setState, getState) => ({
+    'data-hook': 'storybook-TextField',
+    value: '',
+    onChange: ({ target: { value } }) => setState({ value }),
+  }),
+
+  examples: <Examples />,
```

Comment:
Let's try to use the same storybook format as the other components (Textarea) for instance.
Using sections and our custom `settingsPanel` method

Reply:
Done